libraries/engage/locations/components/StoreList/StoreAddressMailingAddress.jsx

```diff
@@ -10,12 +10,14 @@ import { address as container, addressIcon } from './style';
  * @returns {JSX}
  */
 function StoreAddressMailingAddress({ address }) {
+  const mapsUrl = useMemo(() => address && generateGoogleMapsDirectionsUrl(address), [address]);
+
   if (!address) {
     return null;
   }
 
   return (
-    <div className={container}>
+    <Link openExtern href={mapsUrl} className={container}>
```

Comment:
There is console error:
`Failed prop type: Invalid prop `className` of type `object` supplied to `Link`, expected `string`.
    in Link (created by Connect(Link))`